cmd/riverui/main.go

```diff
@@ -78,10 +78,12 @@ func initAndServe(ctx context.Context) int {
 	}
 
 	handlerOpts := &riverui.HandlerOpts{
-		Client: client,
-		DBPool: dbPool,
-		Logger: logger,
-		Prefix: pathPrefix,
+		Client:            client,
+		DBPool:            dbPool,
+		Logger:            logger,
+		Prefix:            pathPrefix,
+		BasicAuthUser:     os.Getenv("BASIC_AUTH_USER"),
+		BasicAuthPassword: os.Getenv("BASIC_AUTH_PASSWORD"),
```

Comment:
You use "user" here, but "username" below.

Do you want to change all references to `user`/`pass`/`BASIC_AUTH_USER`/`BASIC_AUTH_PASS`? The symmetry in length appeals to me, and it's a little shorter.

Also, can you keep this list alphabetized?